setup.py

```diff
@@ -30,6 +30,10 @@ def find_stub_files(name: str) -> List[str]:
     "types-PyYAML",
 ]
 
+extras_require = {
+    "compatible-mypy": ["mypy<0.970"],
```

Comment:
```suggestion
    "compatible-mypy": ["mypy>=0.930,<0.970"],
```

Reply:
Not strictly necessary as `extras_require` lists are always merged with `install_requires`, but if you prefer this, sure.